crates/l2/proposer/l1_committer.rs

```diff
@@ -20,7 +20,7 @@ use ethrex_l2_sdk::{
     eth_client::{eth_sender::Overrides, BlockByNumber, EthClient, WrappedTransaction},
 };
 use ethrex_storage::{error::StoreError, Store};
-use ethrex_vm::{evm_state, execute_block, get_state_transitions};
+use ethrex_vm::{db::evm_state, evm_backends::revm::execute_block, get_state_transitions};
```

Comment:
there should be nothing referencing revm or levm outside `vm` crate, besides the enum

Reply:
Sounds good!